src/lib.rs

```diff
@@ -31,6 +31,7 @@ extern crate multipart;
 extern crate base64;
 extern crate flate2;
 extern crate serde;
+pub extern crate chrono;
```

Comment:
Sure thing!  In my own crates if I use another Crate's structures I prefer to re-export to save users the effort of finding the version I used, but if you'd prefer to not re-export I'll change this to do that.